bower.json

```diff
@@ -26,14 +26,5 @@
     "Gruntfile.js",
     "karma.conf.js",
     "package.json"
-  ],
-  "dependencies": {
-    "d3": "~3.5.6"
-  },
-  "resolutions": {
-    "d3": "~3.5.6"
-  },
-  "devDependencies": {
-    "bootstrap-sass-official": "~3.3.4"
-  }
+  ]
```

Comment:
We still need the dependencies here, otherwise when others bower install this, they won't be included. Make sure they match the deps in package.json and I _think_ you can remove resolutions.... you'll have to do a local [link](http://bower.io/docs/api/#link) and test it out.